docs/rst/manual/reference/configuration/keywords.rst

```diff
@@ -2081,26 +2135,155 @@ Configuring the RSH queue (deprecated)
 
 	::
 
-		RSH_COMMAND /usr/bin/ssh
+		QUEUE_OPTION RSH RSH_COMMAND /usr/bin/ssh
 
 
 .. _max_running_rsh:
-.. topic:: MAX_RUNNING_RSH
+.. topic:: MAX_RUNNING
 
-	The keyword MAX_RUNNING_RSH controls the maximum number of simultaneous jobs
-	running when using the RSH option in QUEUE_SYSTEM. If MAX_RUNNING_RSH exceeds
-	the total capacity defined in RSH_HOST, it will automatically be truncated to
+	The queue option keyword MAX_RUNNING controls the maximum number of simultaneous
+	jobs running when (in this case) using the RSH option in QUEUE_SYSTEM. If MAX_RUNNING
+	exceeds the total capacity defined in RSH_HOST, it will automatically be truncated to
 	that capacity.
 
 	*Example:*
 
 	::
 
+		QUEUE_SYSTEM RSH
 		-- No more than 10 simultaneous jobs
 		-- running via RSH.
-		MAX_RUNNING_RSH 10
+		QUEUE_OPTION RSH MAX_RUNNING 10
```

Comment:
Is this the correct syntax for `MAX_RUNNING` in general? If so, I think the `MAX_RUNNING` docs needs to be updated also...

Reply:
And otherwise we should add an issue about making the configuration of `MAX_RUNNING` similar over the various queue systems!

Reply:
I have to admit, I have not tested the configuration on the various clusters!

But, unless this is silently ignored, or the validation hits later, the following configurations are all valid (not the values though):

```
QUEUE_OPTION RSH MAX_RUNNING 10
QUEUE_OPTION LSF MAX_RUNNING 10
QUEUE_OPTION SLURM MAX_RUNNING 10
```
From what I can see in the code, this should be the correct way to set it up. The `MAX_RUNNING` is a keyword defined in the top level.

Reply:
The `MAX_RUNNING` option was added to all but the LSF, I've added it there as well

Reply:
>If so, I think the MAX_RUNNING docs needs to be updated also...

Not sure where that is? I think there used to be entries for each of the queues with `MAX_RUNNING_LSF` etc, but not a single `MAX_RUNNING` entry.